MediaAdmin/FileAlternatives/Strategy/AbstractFileAlternativesStrategy.php

```diff
@@ -10,11 +10,22 @@
  */
 abstract class AbstractFileAlternativesStrategy implements FileAlternativesStrategyInterface
 {
+    const MEDIA_TYPE = "abstract";
+
     protected $mediaStorageManager;
     protected $tmpDir;
     protected $fileSystem;
 
     /**
+     * Get the $media type supported by the strategy
+     *
+     * @return srtring
+     */
+    public function getMediaType() {
+        return static::MEDIA_TYPE;
```

Comment:
i think is not a good idea to specify a abstract media. but just add an abstract method getMediaType to forced an integrator to specify a media type